templates/automotive/index.html

```diff
@@ -220,43 +65,348 @@ <h2 class="p-muted-heading u-align--center u-sv3">THESE AUTOMOTIVE INNOVATORS HA
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/b2b4d912-community-support-thin.svg",
+          alt="",
+          height="200",
+          width="160",
+          hi_def=True,
+        ) | safe
+      }}
+      <h2 class="p-heading--five">Open source 2.0</h2>
+      <p class="p-heading--two">Better tools. Bigger ecosystem. Broader talent pool.</p>
+      <p>Leaders choose Ubuntu because <a href="https://medium.com/@olley_io/what-software-do-autonomous-vehicle-engineers-use-part-1-2-275631071199" class="p-link--external">autonomous vehicle engineers are more productive with it</a>. From cloud to edge.</p>
+      <p>Develop on Ubuntu today and pick your platform later.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-6">
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/c53664cd-Security_updates.svg",
+          alt="",
+          height="200",
+          width="335",
+          hi_def=True,
+        ) | safe
+      }}
+      <h2 class="p-heading--five">Safety first</h2>
+      <p class="p-heading--two">Top rated security. Best-in-class isolation.</p>
+      <p>We power innovation. And production.</p>
+      <p><a href="/esm">Ten year CVE security maintenance</a>, the most comprehensive <a href="/embedded"></a>container confinement and kernel livepatch</a> for rapid rollouts in critical situations.</p>
+      <p>Because safety depends on the ability to evolve, resist and respond to new threats.</p>
+    </div>
+    <div class="col-6">
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/f880a3bd-Enterprise+support.svg",
+          alt="",
+          height="200",
+          width="200",
+          hi_def=True,
+        ) | safe
+      }}
+      <h2 class="p-heading--five">Long term support</h2>
+      <p class="p-heading--two">Extended security, extended opportunity</p>
+      <p>You innovate. We maintain.</p>
+      <p>Extend ECU life and lifetime value with platform upgrades for new services and solutions.</p>
+      <p>Count on Canonical security coverage. Regular updates. Automated testing. Continuous deployment. Progressive releases.</p>
+      <p>A new world of constant car improvement.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-6">
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/1d5f293c-compliance.svg",
+          alt="",
+          height="200",
+          width="330",
+          hi_def=True,
+        ) | safe
+      }}
+      <h2 class="p-heading--five">Open source licensing</h2>
+      <p class="p-heading--two">Straightforward compliance</p>
+      <p>Confidently integrate open source components into your proprietary solutions.</p>
+      <p>Source identification for all software components, all the time.</p>
+      <p>Simplify compliance. Streamline certification. Protect innovation.</p>
+    </div>
+    <div class="col-6">
```

Comment:
all four of the images in this section are far bigger than the design

Reply:
Yeah, I was going to wait for the logos in the "ARM and X86" section to get the correct height.